app/filter-helpers.js

```diff
@@ -10,19 +10,60 @@ for (var group in formatMappingData) {
 
 var sectionFilter = function (allContent, pageSection) {
   console.log("Filtering " + allContent.length + " items by " + pageSection);
+  var filteredList = _filteredList(allContent, pageSection);
+
+  console.log("Filtered out " + filteredList.length);
+  console.log(filteredList);
+
+  return filteredList;
+}
+
+var rowsWithThreeColumns = function(allContent, pageSection) {
```

Comment:
Come to think of it, weren't we aiming for a pure CSS solution for this grid thing?